src/components/pager/pager.component.js

```diff
@@ -22,11 +22,19 @@ const Pager = (props) => {
   }, [props.currentPage]);
 
   /** Term used to describe table data */
-  const descriptor = I18n.t(
+  const records = count => I18n.t(
     'pager.records',
     {
-      count: Number(props.totalRecords),
-      defaultValue: isClassic(currentTheme) ? ' records' : ' items'
+      count: Number(count),
+      defaultValue: isClassic(currentTheme) ? {
+        one: 'record',
+        zero: 'records',
+        other: 'records'
```

Comment:
@jamime is there a reason why you have to define both `zero` and `other`?

Reply:
Removed.